pagopa/io-app
# chore: [IOBP-527] Remove require cycles in CGN

## Short description
removed require cycles in CGN folder

## List of changes proposed in this pull request
- moved circular imports to shared files


## How to test
`yarn jest` and `yarn tsc:noemit` should be enough; 
The existance of circular dependencies has been checked [this VSCode extension]( https://marketplace.visualstudio.com/items?itemName=iulian-radu-at.find-unused-exports), along with VSCode's very useful search in non-edit views feature.
